env_setup.sh

```diff
@@ -0,0 +1,15 @@
+#/bin/bash
+
+PYTHON=$1
+VENV_PATH=$2
+
+$PYTHON -m venv $VENV_PATH
+source $VENV_PATH/bin/activate
+pip install --upgrade pip
+
+for line in $(grep -v \# requirements.txt)
```

Comment:
What's wrong with `pip install -r requirements.txt`?

Reply:
That does not install packages in order.